Packs/Mattermost/Integrations/MattermostV2/MattermostV2.yml

```diff
@@ -48,6 +50,26 @@ configuration:
   section: Collect
   advanced: true
   required: false
+- display: Types of Notifications to Send
+  name: permitted_notifications
+  type: 16
+  section: Collect
```

Comment:
note that this one is "Collect" and notification_channel is "Connect"

Reply:
fixed